src/solver/conv_direct_naive_conv_bwd.cpp

```diff
@@ -163,6 +165,14 @@ ConvSolution ConvDirectNaiveConvBwd::GetSolution(const ExecutionContext& ctx,
                 auto out_strides =
                     conv_internal::MakeStrideArray<5>(conv_internal::SplitStrideCtoGC(
                         group, tensors.outDesc.GetStrides(), G_stride_idx));
+                float alpha_val = 1.0;
```

Comment:
We cannot assume `float` as a data type for `alpha/beta`. See: https://docs.nvidia.com/deeplearning/cudnn/latest/developer/misc.html#scaling-parameters